source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,268 @@
+=======
+Indexes
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Indexes are data structures that support the efficient execution of queries in MongoDB. They contain copies of parts of
+the data in documents to make queries more efficient.
+
+Without indexes, MongoDB must scan *every* document in a collection to find the documents that match each query. These
+collection scans are slow and can negatively affect the performance of your application. By using an index to limit the
+number of documents MongoDB scans, queries can be more efficient and return faster.
+
+
+Query Coverage and Performance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you execute a query against MongoDB, your query can include three parts:
```

Comment:
I'm concerned that this description is getting ahead of what we say on https://docs.mongodb.com/manual/indexes/#index-use.  And it's incomplete as it doesn't cover, for example, geospatial indexes.

Reply:
We'll have the query team lgtm the "common" parts as delimited in the rst